infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -1012,6 +1017,26 @@ private boolean verify(List<LogData> range) {
         return endSegment - firstSegment <= 1;
     }
 
+    /**
+     * This method requests for missing addresses in this Log Unit in the specified consecutive
+     * range of addresses.
+     *
+     * @param range Range of addresses to compare.
+     * @return List of missing addresses.
+     */
+    @Override
+    public List<Long> getMissingAddresses(Range<Long> range) {
+
+        List<Long> result = new ArrayList<>(10_000);
+        for (long address = range.lowerEndpoint(); address <= range.upperEndpoint(); address++) {
```

Comment:
I think the converse is more natural. Given a set, we have contains method and not doesntContain.